dist/t/spec/features/login_spec.rb

```diff
@@ -75,19 +75,46 @@
   end
 
   it "should be able to check Build Results and see succeeded package built" do
+    sleep(10)
     visit "/package/live_build_log/home:Admin/obs-build/openSUSE_Tumbleweed/i586"
-    wait_for_ajax
+    begin
+      Timeout.timeout(160) {
+        if page.has_content?("No live log available:") then
+          page.evaluate_script("window.location.reload()")
+          first(:link, "Start refresh").click
+        end
+        expect(page).to have_selector("div#log_space_wrapper", :wait => 20)
+        expect(page).to have_content('finished "build build.spec"', :wait => 160)
+      }
+    rescue Timeout::Error
+      puts "Let's retry on rescue."
```

Comment:
if the job fails in openQA, i'm uploading /srv/[obs|www] , but yeah we can put it on file